apps/api/controllers/userController.ts

```
@@ -83,26 +90,26 @@ export async function findConnectionByUserId(req: Request, res: Response) {
  * Responds with the user object as JSON. Returns a 400 error if the email is missing from the request body, or a 500 error if the operation fails.
  */
 export async function findOrCreateUser(req: Request, res: Response) {
-  try {
-    const externalId = req?.userId!;
-    const { email, imageUrl } = req?.body;
+    try {
+        const externalId = req?.userId!;
+        const { email, imageUrl } = req?.body;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)